src/dst/expr.cpp

```diff
@@ -19,58 +19,64 @@
 #define _XOPEN_SOURCE 700
 #define _POSIX_C_SOURCE 200809L
 
+#include "expr.h"
+
 #include <cassert>
 #include <sstream>
 
 #include "util/diagnostic.h"
-#include "expr.h"
 
 CPPFile expr_h("src/dst/expr.h");
 static CPPFile cppFile(__FILE__);
 
-Expr::~Expr() { }
-const TypeDescriptor Prim      ::type("Prim");
-const TypeDescriptor App       ::type("App");
-const TypeDescriptor Lambda    ::type("Lambda");
-const TypeDescriptor VarRef    ::type("VarRef");
-const TypeDescriptor Literal   ::type("Literal");
+Expr::~Expr() {}
+const TypeDescriptor Prim ::type("Prim");
+const TypeDescriptor App ::type("App");
+const TypeDescriptor Lambda ::type("Lambda");
+const TypeDescriptor VarRef ::type("VarRef");
+const TypeDescriptor Literal ::type("Literal");
```

Comment:
Am I crazy or would clang-format not add a space if we removed the space here?

Reply:
Would you like be to go ahead and remove the extra space while I'm here?

Reply:
If you want to that's fine. I don't really think it makes sense for us to nit pick over issues like this. We can fix those as we edit the relevant files.

We just need a policy about allowing formatting fixes to be done as we go with other PRs

Reply:
Makes sense. I think we will have to write the policy as we explore the code change over the next several PRs

One to start with

Replace `X ::Y()` with `X::Y()`